<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>985</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeNetworkAoALocator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>621</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Locators</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>601</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Modify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>601</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Locator et data</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>790</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>MQTT Broker link</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>116</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Subscribe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>publish</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connected</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>List Asset visible</string>
    </property>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>601</width>
      <height>941</height>
     </rect>
    </property>
    <property name="title">
     <string>Affichage 2D</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>541</width>
       <height>811</height>
      </rect>
     </property>
     <property name="interactive">
      <bool>true</bool>
     </property>
     <property name="sceneRect">
      <rectf>
       <x>0.000000000000000</x>
       <y>0.000000000000000</y>
       <width>0.000000000000000</width>
       <height>0.000000000000000</height>
      </rectf>
     </property>
     <property name="rubberBandSelectionMode">
      <enum>Qt::IntersectsItemBoundingRect</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Les unit√©es sont en Metres</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>940</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fermer le programme</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>670</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>ADD TO LOG:</string>
    </property>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SAVE IN A CSV</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>&amp;tools</string>
    </property>
    <addaction name="action_Options"/>
    <addaction name="action_Fermer"/>
   </widget>
   <widget class="QMenu" name="menu_info">
    <property name="title">
     <string>&amp;info</string>
    </property>
    <addaction name="action_info"/>
   </widget>
   <addaction name="menu_tools"/>
   <addaction name="menu_info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="action_Fermer">
   <property name="text">
    <string>&amp;Fermer</string>
   </property>
  </action>
  <action name="action_info">
   <property name="text">
    <string>&amp;info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
